src/schema/schema.rs

```diff
@@ -358,6 +383,21 @@ impl Schema {
         }
         Ok(doc)
     }
+
+    /// Searches for a full_path, returning the field name and a json path.
```

Comment:
This is a public method in a library.

Can you elaborate the comment here?
What happens if there is a non-empty json path, but the field is not a json field for instance?